common/script_values/wc_magic_perks_values.txt

```diff
@@ -0,0 +1,26 @@
+﻿wc_magic_lifestyle_focus_auto_selection_weight_base_value = 200
+wc_magic_lifestyle_focus_auto_selection_weight_add_to_base_value = 1800
+wc_magic_lifestyle_perk_focus_selection_weight_multiply_common_value = {
+	value = 1
+	if = {
+		limit = {
+			OR = {
+				has_trait = magic_bad
+				has_trait = magic_bad_noninheritable
+			}
+		}
+		multiply = 0
+	}
+	if = {
+		limit = { has_trait = education_martial }
+		multiply = 1.25
+	}
+	else_if = {
+		limit = { has_trait = education_learning }
+		multiply = 2
+	}
+}
+
+wc_magic_lifestyle_perk_auto_selection_weight_base_value = 11
+wc_magic_lifestyle_perk_auto_selection_weight_add_to_base_value = 1989
```

Comment:
I don't get the point of this math. So the base value of picking focus (that gives you exp) is higher than the base value of picking perks (for exp you just got from the focus). So the AI picks the focus but doesn't pick perks? This part needs in-file documentation.

Reply:
In vanilla, it's one value, 11 and 1989 for having the right education.

Reply:
If it's due to `education_martial`, `education_learning` multipliers. Don't use them as multipliers.

Reply:
> So the AI picks the focus but doesn't pick perks?

AI pick focus more often regardless of other factors.
If AI picks the focus, it will pick perks from this lifestyle much more likely than from others.

> In vanilla, it's one value, 11 and 1989 for having the right education.

In vanilla AI have base value of 2000 for their education, 11 for each other focuses (2044 total for vanilla focuses)

So, depends on amount of magic trees available, AI has base chance of 10%(value of 200 with 1 magic allowed)-50%(10 values of 200 with 10 magic allowed) to choose one of magical focuses.

Reply:
I'll think about how to change it if you don't like how it's works

Reply:
> So, depends on amount of magic trees available, AI has base chance of 10%(value of 200 with 1 magic allowed)-50%(10 values of 200 with 10 magic allowed) to choose one of magical focuses.

Why diplomats and stewards pick magic? This percent is too high.

Reply:
It makes much more sense to keep the base low and add an education add bonus instead of the multiplier.

Reply:
Done